Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,88 @@
+Enable Hubble
+==============
+
+Hubble is a fully distributed networking and security observability platform
+for cloud native workloads. It is built on top of Cilium and eBPF to enable
+deep visibility into the communication and behavior of services as well as the
+networking infrastructure in a completely transparent manner.
+
+* Specify the namespace in which Cilium is installed as ``CILIUM_NAMESPACE``
+  environment variable. Subsequent commands reference this environment variable.
+
+  .. parsed-literal::
+
+      export CILIUM_NAMESPACE=kube-system
+
+  .. note::
+
+      Cilium is typically installed in ``kube-system`` namespace, but there are
+      a few exceptions such as GKE which install Cilium in a dedicated ``cilium``
```

Comment:
```suggestion
      a few exceptions such as GKE which installs Cilium in a dedicated ``cilium``
```

Reply:
i googled this one :)

https://ell.stackexchange.com/questions/9684/a-few-takes-a-singular-or-plural-verb-in-present-simple-tense